src/__tests__/server.test.ts

```diff
@@ -1,4 +1,49 @@
-import { Express, Request, Response } from "express";
+// tslint:disable-next-line: no-object-mutation
+process.env = {
+  ...process.env,
+  ALLOW_NOTIFY_IP_SOURCE_RANGE: "::ffff:ac13:1/112",
+  ALLOW_PAGOPA_IP_SOURCE_RANGE: "::ffff:ac13:1/112",
+  API_BASE_PATH: "/api/v1",
+  API_KEY: "put_your_api_key_here",
+  API_URL: "http://functions:7071/api/v1",
+  AUTHENTICATION_BASE_PATH: "",
+  AZURE_NH_ENDPOINT:
+    "Endpoint=sb:// io-lab-notification-hub-ns.servicebus.windows.net/;SharedAccessKeyName=DefaultFullSharedAccessSignature;SharedAccessKey=C4xIzNZv4VrUnu5jkmPH635MApRUj8wABky8VfdPLqg=",
+  AZURE_NH_HUB_NAME: "put_nh_hub_name_here",
+  AzureWebJobsStorage:
+    // tslint:disable-next-line: no-duplicate-string
+    "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://fnstorage:10000/devstoreaccount1;QueueEndpoint=http:// fnstorage:10001/devstoreaccount1;TableEndpoint=http://fnstorage:10002/devstoreaccount1;",
+  CIE_METADATA_URL:
+    "https://idserver.servizicie.interno.gov.it:8443/idp/shibboleth",
+  CLIENT_ERROR_REDIRECTION_URL: "/error.html",
+  CLIENT_REDIRECTION_URL: "/profile.html?token:",
+  IDP_METADATA_URL:
+    "https://registry.spid.gov.it/metadata/idp/spid-entities-idps.xml",
+  LogsStorageConnection:
+    "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://fnstorage:10000/devstoreaccount1;QueueEndpoint=http:// fnstorage:10001/devstoreaccount1;TableEndpoint=http://fnstorage:10002/devstoreaccount1;",
```

Comment:
why do we need all of this configuration ?

Reply:
Because, while initializing newApp in order to simulate graceful shutdown, tests run failed due to required env variables not found ( in `config.ts` )

Reply:
how come this test was working before this changes ?

Reply:
On my local environment (running yarn test) it just fails!

Reply:
please remove this code here and try to run tests using `yarn test:coverage` as exposed into the pipeline :)